README.md

```diff
@@ -16,53 +16,51 @@
 
 </div>
 
-## Usage
-
-First, go to the repository you want to check.
+## Configuration
 
+This tool looks for configuration in a `pyproject.toml`file in the root of the codebase being analysed.
 
-### [black](https://black.readthedocs.io/en/stable/)
+The default configuration values are:
 
-Check **src** directory and **setup.py** file with **black**
-```bash
-docker run --rm -v "${PWD}":/github/workspace plone/code-quality:latest check black src setup.py
+```toml
+[tool.plone-code-analysis]
+checkers = ["black", "flake8", "isort", "pyroma", "zpretty"]
+formatters = ["black", "isort", "zpretty"]
+paths = "."
+paths_pyroma = "."
+paths_black = "."
+paths_flake8 = "."
+paths_isort = "."
+paths_pyroma = "."
+paths_zpretty = "."
 ```
 
-### [flake8](https://flake8.pycqa.org/en/stable/)
+If you want to change only the `paths`, you should add to your `pyproject.toml` the following settings:
 
-Flake8 checks, using [flakeheaven](https://pypi.org/project/flakeheaven/) configuration format.
-
-Current plugins in use:
-
-* [flake8-blind-except](https://pypi.org/project/flake8-blind-except/)
-* [flake8-debugger](https://pypi.org/project/flake8-debugger/)
-* [flake8-print](https://pypi.org/project/flake8-print/)
-
-Check **src** directory and **setup.py** file with **flake8**
-
-```bash
-docker run --rm -v "${PWD}":/github/workspace plone/code-quality:latest check flake8 src setup.py
+```toml
+[tool.plone-code-analysis]
+paths = "src/ setup.py"
```

Comment:
And I did the image build and added the following configuration in my `pyproject.toml`:

```toml
[tool.plone-code-analysis]
checkers = ["black", "flake8", "isort", "pyroma", "zpretty"]
formatters = ["black", "isort", "zpretty"]
paths = "src/"
```

then I ran:

```
docker run --rm -v "${PWD}":/github/workspace plone/code-quality:latest check isort
```

I was expecting it to only check the `src` folder. But it started complaining about files that were inside the `lib` folder:

```
ERROR: /github/workspace/lib/python3.10/site-packages/plone/app/theming/tests/test_exportimport.py Imports are incorrectly sorted and/or formatted.
```

Reply:
I will add an environment flag to explicitly report which paths are being analysed.

Reply:
@wesleybl Now it is possible to run this image with a more verbose logging. could you try (after building it locally):

```shell
docker run -e LOG_LEVEL=DEBUG --rm -v "${PWD}":/github/workspace plone/code-quality:latest check isort
```
and paste the log here?

Reply:
It took considerable time and then returned this:

```bash
$ docker run -e LOG_LEVEL=DEBUG --rm -v "${PWD}":/github/workspace plone/code-quality:latest check isort

============================
Starting plone-code-analysis
============================
Provided paths [/github/workspace]
Settings: {'black': [src], 'flake8': [src], 'isort': [src], 'pyroma': [/github/workspace], 'zpretty': [src]}

Running check: isort
====================
Running command ['isort', '--check-only', /github/workspace]

Skipped 4 files


=======
Results
=======
There were errors
```

The code has no isort errors

Reply:
@wesleybl For some reason it is not finding your `pyproject.toml` (and using the default path).are you sure the file is in the root of the mounted folder?

Reply:
@wesleybl I just pushed a new version, could you please try again?

Reply:
> For some reason it is not finding your pyproject.toml (and using the default path)

I don't think that's it. The file is there:

```bash
$ ls "${PWD}"
CHANGES.rst       docs         MANIFEST.in     README.rst
CONTRIBUTORS.rst  LICENSE.rst  news            setup.py
DEVELOP.rst       Makefile     pyproject.toml  src
```
When I rename it, it doesn't take long, and the result is different:

```
$ mv pyproject.toml pyproject.toml.old
$ docker run -e LOG_LEVEL=DEBUG --rm -v "${PWD}":/github/workspace plone/code-quality:latest check isort

============================
Starting plone-code-analysis
============================
File not found: pyproject.toml
Provided paths [/github/workspace]
Settings: {'black': [], 'flake8': [], 'isort': [], 'pyroma': [], 'zpretty': []}

Running check: isort
====================
Running command ['isort', '--check-only', /github/workspace]
Skipped 1 files


=======
Results
=======
There were errors
```

Reply:
Looks like I missed completing `/github/workspace` with `/src`

Reply:
I believe I found the issue. When running a single check, without explicitly setting the paths, `docker-entrypoint.py` assumed the paths to be `./`.

@wesleybl, could you try it again and see if it is fixed?

Reply:
@ericof It worked. but when it has error, it doesn't show where it is:

```bash
$ docker run --rm -v "${PWD}":/github/workspace plone/code-quality:latest check isort

============================
Starting plone-code-analysis
============================

Running check: isort
====================


=======
Results
=======
There were errors
```

Reply:
Checking that

Reply:
Fixed the issue on the last commit

Reply:
Now it's working! Thanks for your work!